js/dlg/message-box.js

```diff
@@ -149,11 +175,27 @@ messageBox.show = async ({
       window.on('scroll', messageBox.listeners.scroll, {passive: false});
     }
     window.on('keydown', messageBox.listeners.key, true);
+    window.on('mouseup', messageBox.listeners.mouseUp, {passive: true});
```

Comment:
I would like for these to be registered inside mouseDown() and unregistered inside mouseUp(). It would allow to remove `if (dragging)`.